app/main/views/services.py

```diff
@@ -113,6 +105,77 @@ def update_service_status(service_id):
     )
 
 
+@main.route(
+    '/services/<string:service_id>/edit/<string:section>',
+    methods=['GET']
+)
+@login_required
+@flask_featureflags.is_active_feature('EDIT_SERVICE_PAGE')
+def edit_section(service_id, section):
+
+    service = data_api_client.get_service(service_id)['services']
+
+    if not _is_service_associated_with_supplier(service):
+        abort(404)
+
+    return render_template(
+        "services/edit_section.html",
+        section=content.get_section_filtered_by(section, service),
+        service_data=service,
+        service_id=service_id,
+        **main.config['BASE_TEMPLATE_DATA']
+    )
+
+
+@main.route(
+    '/services/<string:service_id>/edit/<string:section>',
+    methods=['POST']
+)
+@login_required
+@flask_featureflags.is_active_feature('EDIT_SERVICE_PAGE')
+def update_section(service_id, section):
+    service = data_api_client.get_service(service_id).get('services')
+
+    if not _is_service_associated_with_supplier(service):
+        abort(404)
+
+    posted_data = dict(
+        list(request.form.items()) + list(request.files.items())
+    )
+
+    posted_data.pop('csrf_token', None)
```

Comment:
I had a feeling that @allait had it set up so that the csrf token got checked and popped for all POST requests before being processed by the view.  But I can't seem to find that code now - did I dream it, or has it been removed?